Pkgfile

```diff
@@ -132,14 +132,14 @@ vars:
   musl_sha512: 498ec5d7941194a8806f4d42f0f6d218c862996ef1398b737d0d06995e0b7a6574b240a48088f6b84016b14b2776fe463f829dcb11149cdfc1023d496b235c55
 
   # renovate: datasource=github-releases depName=nvidia/open-gpu-kernel-modules
-  nvidia_driver_version: 535.54.03
-  nvidia_driver_sha256: 49ed408fc8bad50990fcc094fe3808975edd81490ac3c36be17a987c7cea4ee7
-  nvidia_driver_sha512: 23fbaaa671f6f4bd92147c1ec0099cd006efc73297211ffba74d78310c14364de726f84e388c397716c42c854a302948f5bcf9e9bac65ae41f54d34e814cfb60
+  nvidia_driver_version: 535.113.01
```

Comment:
let's not bump this, unless we are sure there's a fabricmanager release matching the same: https://github.com/siderolabs/extensions/blob/main/nvidia-gpu/vars.yaml#L1